src/main/java/org/opensearch/security/action/apitokens/ApiToken.java

```diff
@@ -109,6 +152,7 @@ public static ApiToken fromXContent(XContentParser parser) throws IOException {
         List<String> clusterPermissions = new ArrayList<>();
         List<IndexPermission> indexPermissions = new ArrayList<>();
         Instant creationTime = null;
+        long expiration = Long.MAX_VALUE;
```

Comment:
should this be Long.MAX_VALUE?

Reply:
It shouldn't matter here, since when the api token is created and expiration is not specified it defaults to 30 days from creation time. But I changed it to 0 to feel safer in: [08e0890](https://github.com/opensearch-project/security/pull/4967/commits/08e08900b914f7df626692108b8099a745343bee)